src/data_models/process_models/process_start_request_payload.ts

```diff
@@ -1,21 +1,21 @@
 /**
- * Describes the payload that a process model can be started with.
+ * Describes the payload that a ProcessModel can be started with.
  */
 export class ProcessStartRequestPayload {
   /**
-   * Optional: If set, the process engine will use this as the id for the
-   * correlation in which the process model will be executed.
-   * If not provided, the process engine will generated a correlation id by
+   * Optional: If set, the process engine will use this as the ID for the
```

Comment:
```suggestion
   * Optional: If set, the Process-Engine will use this as the ID for the
```

Process-Engine is still our product and it is its name.

Reply:
Or if you want to reference a generic engine, use 

> workflow engine

instead.

Reply:
missed that one.